docs/concepts/fields.md

```diff
@@ -949,7 +953,7 @@ class Box(BaseModel):
     depth: float
 
     @computed_field
-    @property  # (1)!
+    @property
```

Comment:
Already present in the previous code example, no need to add the annotation twice.